core-services/workflow/src/main/java/org/silverpeas/core/workflow/engine/error/WorkflowErrorImpl.java

```diff
@@ -55,16 +95,12 @@ public WorkflowErrorImpl() {
   public WorkflowErrorImpl(ProcessInstance instance, GenericEvent event,
       HistoryStep step, Exception exception) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "step". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1172)